docker/config/local_dev.env

```diff
@@ -25,7 +25,7 @@ CRASHMOVER_CRASHPUBLISH_SUBSCRIPTION_NAME=local_dev_socorro_sub
 PUBSUB_EMULATOR_HOST=pubsub:5010
 
 # GCS settings
-CRASHMOVER_CRASHSTORAGE_GCS_BUCKET_NAME=antennabucket
+CRASHMOVER_CRASHSTORAGE_BUCKET_NAME=antennabucket
```

Comment:
This is repeated on line 39. On the one hand, it makes sense to have S3 and GCS sections. On the other hand, it gets confusing if you change this one and the next one overrides it.

I'm not sure what to do about this short of having separate name spaces, but that gets complicated with how Antenna is structured configuration-wise. :(

Reply:
i'll delete this one, and add a comment on the next one:
```diff
 CRASHMOVER_CRASHSTORAGE_SECRET_ACCESS_KEY=foo
+# Used for S3 and GCS
 CRASHMOVER_CRASHSTORAGE_BUCKET_NAME=antennabucket
```